docs/spec.yaml

```diff
@@ -1096,3 +1490,48 @@ definitions:
         description: |
           For when action type is 'clone' - the name of the environment you wish
           to clone
+  Policy:
+    type: object
+    required:
+      - name
+      - definition
+    properties:
+      id:
+        type: integer
+        description: policy id
+        readOnly: true
+      name:
+        type: string
+        description: the name of the policy
+      description:
+        type: string
+        description: the spec to be processed
+      environments:
+        type: array
+        items:
+            $ref: '#/definitions/Environment'
+  Notification:
+    type: object
+    required:
+      - name
+      - type
+      - config
+      - members
+    properties:
+      id:
+        type: integer
+        description: notification id
+        readOnly: true
+      name:
+        type: string
+        description: the name of the notification
+      type:
+        type: string
+        description: the type of notification
+      config:
+        type: string
+        description: the config required by the notification
+      members:
+        type: array
```

Comment:
members is a type of "string" on the notifications model, should this be corrected?